snippets/reverseString.md

```diff
@@ -2,10 +2,10 @@
 
 Reverses a string.
 
-Use array destructuring and `Array.reverse()` to reverse the order of the characters in the string.
+Use `split('')` and `Array.reverse()` to reverse the order of the characters in the string.
 Combine characters to get a string using `join('')`.
 
 ```js
-const reverseString = str => [...str].reverse().join('');
+const reverseString = str => str.split('').reverse().join('');
```

Comment:
Maybe we should just avoid `split` vs `spread` and make it obvious what we want to do with `Array.from`
```js
Array.from(str).reverse.join('')
```